assembly/pom.xml

```diff
@@ -193,19 +205,31 @@
       <dependencies>
         <dependency>
           <groupId>org.apache.spark</groupId>
-          <artifactId>spark-hive_${scala.binary.version}</artifactId>
+          <artifactId>xsql-hive_${scala.binary.version}</artifactId>
           <version>${project.version}</version>
         </dependency>
+        <dependency>
+          <groupId>org.apache.spark</groupId>
+          <artifactId>spark-hive_${scala.binary.version}</artifactId>
+          <version>${spark.version}</version>
+          <scope>provided</scope>
+        </dependency>
       </dependencies>
```

Comment:
Actually, spark-hive don't need to mark as provided, please remove it.

Reply:
This will lead final assembly jars contains `spark-hive`  we not need.